Lib/bz2.py

```diff
@@ -218,15 +218,22 @@ def readlines(self, size=-1):
     def write(self, data):
         """Write a byte string to the file.
 
-        Returns the number of uncompressed bytes written, which is
-        always len(data). Note that due to buffering, the file on disk
-        may not reflect the data written until close() is called.
+        Returns the number of uncompressed bytes written. Note that
```

Comment:
One detail is missed, present in the old docstring. That all bytes are written.

In the docstring of `BufferedWriter.write`:
```
      Returns the number of bytes written, which is always the length of b in bytes.
```
I think this wording can be borrowed.

Reply:
Added.